### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kfahn22**
Adding some information about Hacktoberfest.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,31 +1,43 @@
-# The Coding Train Website Contributing Guide
+# All Aboard The Coding Train for Hacktoberfest 2022
 
-Thank you for considering contributing to thecodingtrain.com! We appreciate your efforts to maintain the website. This guide will give you an overview of the contribution workflow. Mistakes are always welcome, we're all learning, so please feel free to seek help in case of any confusion, we are here to help.  
+One of the major goals of The Coding Train is to provide a beginner-friendly environment for people to make pull requests, which is why we are participating in [Hacktoberfest](https://hacktoberfest.com/participation/) this year.  Hacktoberfest is DigitalOcean's annual event that encourages people to contribute to open source throughout October.
 
-First, please read our [Code of Conduct](https://github.com/CodingTrain/Code-of-Conduct) to learn about what is and is not acceptable to keep our community respectable, friendly and approachable.
+Here are some key values of Hacktoberfest:
+- Everyone is welcome
+- Quantiy if Fun, Quality is Key
```
Comment (shiffman) [inline CONTRIBUTING.md:7]:
```suggestion
- Quantity is Fun, Quality is Key
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,31 +1,43 @@
...
+- Short-term Act, long term impact
+
+The Coding Train strives to be a respectful, friendly, and approachable community. Read our [Code of Conduct](https://github.com/CodingTrain/Code-of-Conduct) to learn about what is and is not acceptable.
```
Comment (shiffman) [inline CONTRIBUTING.md:10]:
```suggestion
The Coding Train strives to be a respectful, friendly, and welcoming community. Read our [Code of Conduct](https://github.com/CodingTrain/Code-of-Conduct) to learn about what is and is not acceptable.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,31 +1,43 @@
...
 
-Here are the types of contributions we are looking for:
-- Adding your own creative version of a coding challenge or tutorial (passenger showcase)
-- Porting content from the old website
-- Bug-fixes
-- Making the code easier to read (including adding comments)
-- **Add more here...**
+Here are the types of contributions we are looking for this month:
+
+1. Adding your own creative version of a coding challenge or tutorial
```
Comment (shiffman) [inline CONTRIBUTING.md:22]:
```suggestion
1. Share your creative work inspired by a coding challenge or tutorial in the showcase!
```

**Review by shiffman — COMMENTED** [review]
This is great! Made some minor suggestions!

**Review by DenisovichDev — CHANGES_REQUESTED** [review]
Hey! It looks great. I just have a minor suggestion. Putting a space before the bullets make them nest under the ordered list, I think that would look better?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,31 +1,43 @@
...
-Here are the types of contributions we are looking for:
-- Adding your own creative version of a coding challenge or tutorial (passenger showcase)
-- Porting content from the old website
-- Bug-fixes
-- Making the code easier to read (including adding comments)
-- **Add more here...**
+Here are the types of contributions we are looking for this month:
+
+1. Share your creative work inspired by a coding challenge or tutorial in the showcase!
+* You can find more information about submitting a showcase in the [Passenger Showcase Guide](http://thecodingtrain.com/guides/passenger-showcase-guide)
```
Comment (DenisovichDev) [inline CONTRIBUTING.md:23]:
```suggestion
 * You can find more information about submitting a showcase in the [Passenger Showcase Guide](http://thecodingtrain.com/guides/passenger-showcase-guide)
```

**Comment by DenisovichDev** [comment]
I'll add the requested changes below

File: CONTRIBUTING.md (hunk 4 above)
Comment (DenisovichDev) [inline CONTRIBUTING.md:23]:
```suggestion
 * You can find more information about submitting a showcase in the [Passenger Showcase Guide](http://thecodingtrain.com/guides/passenger-showcase-guide)
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,31 +1,43 @@
...
+
+2. Create a page for a Coding Challenge video! (https://github.com/CodingTrain/thecodingtrain.com/issues/155)
+* Head to the [spreadsheet tracking challenges](https://docs.google.com/spreadsheets/d/1zt8KquMZN_j2-j4ezMsgeq-sRrgoCEAROwFS6LTS8oQ/edit#gid=1236591781) and sign up.
```
Comment (DenisovichDev) [inline CONTRIBUTING.md:26]:
```suggestion
 * Head to the [spreadsheet tracking challenges](https://docs.google.com/spreadsheets/d/1zt8KquMZN_j2-j4ezMsgeq-sRrgoCEAROwFS6LTS8oQ/edit#gid=1236591781) and sign up.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,31 +1,43 @@
...
+* For instructions, read the [Maintaining the website guide](/guides/maintain-weibsite-guide/) on how to create a challenge page on the new website.
```
Comment (DenisovichDev) [inline CONTRIBUTING.md:27]:
```suggestion
 * For instructions, read the [Maintaining the website guide](/guides/maintain-weibsite-guide/) on how to create a challenge page on the new website.  
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,31 +1,43 @@
...
-- **Add more here...**
+Here are the types of contributions we are looking for this month:
+
+1. Share your creative work inspired by a coding challenge or tutorial in the showcase!
+ * You can find more information about submitting a showcase in the [Passenger Showcase Guide](http://thecodingtrain.com/guides/passenger-showcase-guide)
+
+2. Create a page for a Coding Challenge video! (https://github.com/CodingTrain/thecodingtrain.com/issues/155)
+ * Head to the [spreadsheet tracking challenges](https://docs.google.com/spreadsheets/d/1zt8KquMZN_j2-j4ezMsgeq-sRrgoCEAROwFS6LTS8oQ/edit#gid=1236591781) and sign up.
+ * For instructions, read the [Maintaining the website guide](/guides/maintain-weibsite-guide/) on how to create a challenge page on the new website.  
+* Note: If you are unable to complete the page, please add a note and remove your name from the spreadsheet.
```
Comment (DenisovichDev) [inline CONTRIBUTING.md:28]:
```suggestion
 * Note: If you are unable to complete the page, please add a note and remove your name from the spreadsheet.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,31 +1,43 @@
...
+
+3. Add Related Challenges! (https://github.com/CodingTrain/thecodingtrain.com/issues/262)
+* The `"relatedChallenges"` property specifies a list of challenges to be featured as suggested on the video page.
```
Comment (DenisovichDev) [inline CONTRIBUTING.md:31]:
```suggestion
 * The `"relatedChallenges"` property specifies a list of challenges to be featured as suggested on the video page.
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,31 +1,43 @@
...
+*  If it is an empty array and there are any coding challenges in the `"video"` section of the json file, those challenge(s) can be added to the field.
```
Comment (DenisovichDev) [inline CONTRIBUTING.md:32]:
```suggestion
 *  If it is an empty array and there are any coding challenges in the `"video"` section of the json file, those challenge(s) can be added to the field.
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,31 +1,43 @@
...
+ - You can find the JSON files for challenges in `/content/videos/challenges`.
+
+4. More!
+* You can find more things to [work on listed in issues](https://github.com/CodingTrain/thecodingtrain.com/issues).
```
Comment (DenisovichDev) [inline CONTRIBUTING.md:36]:
```suggestion
 * You can find more things to [work on listed in issues](https://github.com/CodingTrain/thecodingtrain.com/issues).
```

**Comment by kfahn22** [comment]
Thanks @DenisovichDev,  It does look better!

**Comment by DenisovichDev** [comment]
@kfahn22 sorry again for the mess. For some reason GitHub wasn't registering a single space or tab, had to add four instead. You can merge it now

**Pull request merged**